desktop/apps/artwork/client/index.coffee

```diff
@@ -33,23 +34,29 @@ sharedInit = [
 module.exports =
   setup: setup = (context = {}) ->
     if context.__typename is 'ArtworkContextAuction'
+      queryParams = qs.parse(location.search.replace(/^\?/, ''))
+
       query: """
-          query artwork($id: String!, $isClosed: Boolean!) {
+          query artwork($id: String!, $isClosed: Boolean!, $auctionId: ID) {
             artwork(id: $id) {
               ... partner
               ... auction_artworks @skip(if: $isClosed)
               ... artist_artworks @include(if: $isClosed)
               ... related_artworks @include(if: $isClosed)
             }
+
+            ... followed_artist_ids @skip(if: $isClosed)
           }
           #{require '../../../components/artwork_brick/query.coffee'}
           #{require '../components/partner/query.coffee'}
-          #{require '../components/auction_artworks/query.coffee'}
+          #{require('../components/auction_artworks/query.coffee').auction_artworks}
+          #{require('../components/auction_artworks/query.coffee').followed_artist_ids(CurrentUser.orNull())}
           #{require '../components/artist_artworks/query.coffee'}
           #{require '../components/related_artworks/query.coffee'}
         """
       variables:
         isClosed: context.is_closed
+        auctionId: queryParams.auction_id
```

Comment:
Seems a little odd/dangerous to be grabbing the auction id only from the query param given that you can visit the artwork page without the `?auction_id` specified.